packages/builder/src/components/design/settings/controls/FieldConfiguration/EditFieldPopover.svelte

```diff
@@ -0,0 +1,160 @@
+<script>
+  import { Icon, Popover, Layout } from "@budibase/bbui"
+  import { store } from "builderStore"
+  import { cloneDeep } from "lodash/fp"
+  import { createEventDispatcher } from "svelte"
+  import ComponentSettingsSection from "../../../../../pages/builder/app/[application]/design/[screenId]/[componentId]/_components/Component/ComponentSettingsSection.svelte"
+  import { getContext } from "svelte"
+
+  export let anchor
+  export let field
+  export let componentBindings
+  export let bindings
+
+  const draggable = getContext("draggable")
+  const dispatch = createEventDispatcher()
+
+  let popover
+  let drawers = []
+  let sudoComponentInstance
```

Comment:
As much as I love something called `sudoComponentInstance` I think this should probably be `pseudoComponentInstance` should it? Or maybe even `fake` or `tempt` to make it easier to type.